dash-pipeline/SAI/templates/saiapi.cpp.j2

```diff
@@ -443,22 +443,25 @@ static sai_status_t dash_sai_create_{{ table.name }}(
     // Search the action
     for (uint32_t i = 0; i < attr_count; i++)
     {
-        switch(attr_list[i].value.s32)
+        if (SAI_{{ table.name | upper }}_ATTR_ACTION == attr_list[i].id)
```

Comment:
previously there was switch, and now there is if, and if attribure is not action, then all other attributes are skipped, in previous case, at least it was logged that some attribue is passed, now there is no log message, 

i propose to make this (if) and then at the end make continue, and leave switch statement as is after if.ACTION case